cloud/amazon/ecs_taskdefinition.py

```diff
@@ -196,6 +196,10 @@ def main():
     if module.params['state'] == 'present':
         if 'containers' not in module.params or not module.params['containers']:
             module.fail_json(msg="To use task definitions, a list of containers must be specified")
+        else: 
+            for container in module.params['containers']:
+                container['cpu'] = int(container['cpu'])
+                container['memory'] = int(container['memory'])
```

Comment:
If `cpu` or `memory` is absent from the definition, this will fail with an exception. Can we wrap this with a descriptive message using `module.fail_json` that `cpu` and `memory` are require?